amis/packer_alinux.json

```diff
@@ -174,13 +174,13 @@
       "type" : "shell",
       "inline" : [
         "region=\"{{user `region`}}\"",
-        "curl --retry 3 -L https://${region}-aws-parallelcluster.s3.${region}.amazonaws.com$([ \"${region}\" != \"${region#cn-*}\" ] && echo \".cn\")/archives/chef/chef-install.sh | sudo bash -s -- -v {{user `chef_version`}}"
+        "curl --retry 3 -L https://${region}-aws-parallelcluster.s3.${region}.amazonaws.com$([ \"${region}\" != \"${region#cn-*}\" ] && echo \".cn\")/archives/cinc/cinc-install.sh | sudo bash -s -- -v {{user `chef_version`}}"
```

Comment:
could we add a user variable for the bucket to download from and default to `${region}-aws-parallelcluster`?

Reply:
I have added a variable that can be used to specify the whole URL, please check